static/styles/recent_topics.css

```diff
@@ -297,6 +330,10 @@
                 margin-right: 5px;
                 font-size: 15px;
             }
+
+            .unread_count_pm {
+                margin-right: 53px;
```

Comment:
How is this 53px determined? It's possible we want to do some math with variables like we have in `left-sidebar.css` to compute this value... magic numbers like 53px are rarely a good way to do things.